charts/values.yaml

```diff
@@ -50,7 +50,7 @@ fullnameOverride: ""
 
 # Configure Marklogic Admin Username and Password
 auth:
-  adminUsername: admin
+  adminUsername: ""
```

Comment:
Did you change this by accident? I think we should keep the default username as admin?

Reply:
Not really by accident I just wanted to give the choice to the customer to choose the adminUsername.
I know that for many company using admin as admin user is forbidden.
but we can let it.

Reply:
I've add back the adminUsername in the last commit.